data/mods/TEST_DATA/items.json

```diff
@@ -525,7 +525,7 @@
       "CAMERA",
       "MP3"
     ],
-    "flags": [ "WATCH", "ALARMCLOCK", "USE_UPS", "NO_UNLOAD", "RECHARGE", "NO_RELOAD" ]
```

Comment:
Does removing `RECHARGE` mean it can only be recharged using UPS rather than also being rechargeable using car chargers?

Reply:
I think so. I just updated the test smartphone to match normal smartphone since the two had drfited apart a bit.